Examples/Python/python/acts/examples/odd.py

```diff
@@ -47,7 +56,7 @@ def getOpenDataDetector(odd_dir, mdecorator=None):
         mdecorator = acts.JsonMaterialDecorator(
             rConfig=config,
             jFileName=str(odd_dir / "config/odd-material-mapping-config.json"),
-            level=acts.logging.WARNING,
+            level=customLogLevel(minLevel=acts.logging.WARNING),
```

Comment:
Just a stupid question. Why sometimes the level is called maxLevel and sometimes minLevel ?

Reply:
I wanted to make it more explicit compared to the existing `customLogLevel` (e.g. [here](https://github.com/acts-project/acts/blob/main/Examples/Python/python/acts/examples/reconstruction.py#L875)) if you want to have a lower or upper bound on the log level. not sure how useful it is. if we think we can live without it we can also drop it